src/scenario-share/ScenarioShareModal.tsx

```diff
@@ -100,7 +100,7 @@ const UserList: React.FC<{
   );
 };
 
-const ShareButtonContainer = styled.button`
+const ShareButtonContainer = styled.div`
```

Comment:
@macfarlandian - I'm assuming you wanted a `div` here?  Let me know if there's another HTML element you'd prefer.

Reply:
oh, thank you ... I didn't notice this warning until after merging that PR and haven't gotten around to fixing it.
Because this whole thing is clickable I think we do actually want this to be a button, but we want `ScenarioShareButton` to .... not be a button. and maybe to not be named that anymore

Reply:
For what it's worth this entire `div` is still clickable.  If I add `cursor: pointer;` to the styles, the experience will be identical.

Reply:
yeah I know, it's just to keep the markup more semantic. functionally no difference (except possibly for people using screen readers)

Reply:
I flipped things around in this latest commit to make the inner button a "link" container:  https://github.com/Recidiviz/covid19-dashboard/pull/506/commits/e141cff4985166796a3f58c96a962f3df67b0a28